.github/workflows/node.js.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow will do a clean install of node dependencies, cache/restore them, build the source code and run tests and npm audit.
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x, 16.x]
+        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v2
+      with:
+        node-version: ${{ matrix.node-version }}
+        cache: 'npm'
+    - run: npm ci
+    - run: npm test
```

Comment:
@antonok-edm npm test seems to be failing in this repo, do you know why?

Reply:
I don't see any failures when running locally, what errors are you getting?

Reply:
Ah, I see - the tests expect the uBlock origin source code submodule to be initialized, but it's not getting set up on the CI machines.

Can we add `git submodule init` and `git submodule update` to the steps?

Reply:
@antonok-edm thanks! that fixes the tests locally for me but it still fails due to lack of sufficient test coverage %. any thoughts on what to do about that?